package.json

```diff
@@ -51,7 +51,7 @@
   "bugs": {
     "url": "https://github.com/NebulousLabs/skynet-js/issues"
   },
-  "homepage": "https://nebulouslabs.github.io/skynet-js/",
+  "homepage": "https://github.com/NebulousLabs/skynet-js",
```

Comment:
I think it is supposed to be either https://nebulouslabs.github.io/skynet-js or htps://siasky.net/docs

Reply:
I think it is correct as is:

> `homepage`: The url to the project homepage.

- https://docs.npmjs.com/files/package.json